zproject/default_settings.py

```diff
@@ -149,10 +149,8 @@
 THUMBNAIL_IMAGES = False
 SENDFILE_BACKEND: Optional[str] = None
 
-# Base URL of the Tornado server
-# We set it to None when running backend tests or populate_db.
-# We override the port number when running frontend tests.
-TORNADO_SERVER: Optional[str] = 'http://127.0.0.1:9800'
+TORNADO_PORTS: List[int] = []
```

Comment:
One observation on this TORNADO_PORTS interface is that we may down the line want to allow Tornado to be on another server and thus have a list of network addresses; but I'm not sure it's worth trying to open that can of worms at this point; this is a mostly internal interface, so we can change it later.

Reply:
Yes.  This refactoring was intended to look forward to replacing `TORNADO_PORTS` with `TORNADO_HOSTS`, which is `List[str]` of ip:port, with a back-compat mode of ints being treated as `127.0.0.1:port`.  Doing away with a hardcoded TORNADO_SERVER is necessary to make this transition clear.